supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/GenomicsFactory.scala

```diff
@@ -166,6 +166,12 @@ case class GenomicsFactory(applicationName: String, authMode: GoogleAuthMode, en
         .setNetwork(network)
         .setAccelerators(accelerators)
 
+      if(createPipelineParameters.dockerImageCacheDiskOpt.isDefined) {
+        jobLogger.warn("Docker image cache requested for the job, but the job is being executed by Google " +
+          "Genomics API v2alpha1, while the feature is only supported by from Google Life Scielnces API starting from" +
+          "the version v2beta")
+      }
```

Comment:
Warnings in terra go to sentry. Either you'll have to reconfigure the sentry warnings to ignore this message if/when it goes off, or this should be lower priority than a `warn`.